galactic/distribution.yaml

```diff
@@ -3812,6 +3812,23 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: foxy-devel
     status: maintained
+  ublox_dgnss:
+    release:
+      packages:
+      - ublox_dgnss
+      - ublox_dgnss_node
+      - ublox_ubx_interfaces
+      - ublox_ubx_msgs
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/aussierobots/ublox_dgnss-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/aussierobots/ublox_dgnss.git
+      version: master
```

Comment:
CI here is failing because the source branch listed here is `master`, but that doesn't exist in the source repository.